StitchCore/StitchCore/Source/Core/Services/Services.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Services.swift
+//  StitchCore
+//
+//  Created by Jason Flax on 11/14/17.
+//  Copyright © 2017 MongoDB. All rights reserved.
+//
+
+import Foundation
+
+protocol Service {
+    var client: StitchClient { get }
+    var name: String { get }
+}
+
+public class Services {
+    let client: StitchClient
+
+    func twilio(name: String) -> TwilioService {
```

Comment:
[q] Are you okay with this kind of coupling?

Reply:
I'm going to hide this behind a private flag for now.